backend/src/DTO/tags.model.ts

```diff
@@ -6,12 +6,13 @@ export interface subDefaultTag {
   login: string;
   content: string;
   superContent: string;
-  visibility: "public" | "private";
+  visibility: 'public' | 'private';
 }
 
 export interface superDefaultTag {
   id: number;
   content: string;
   login: string;
   count: number;
+  type: string;
```

Comment:
```suggestion
  type: 'super' | 'default';
```
type 값도 마찬가지로 가능한 경우를 명시해주면 좋겠네요!